### ISSUE TITLE
Create character concept art

### ISSUE THREAD

**Issue by aki-cat** [uspgamedev/boomscience#12]

**Comment by MagykalMystique** [comment]
So does this issue still need to be addressed? If you still need help with elaborating on the character art I can help. I'm still quite inexperienced with coding, I've only been doing it for a few months in python, and recently started working on GML, but art is definitely a strength of mine :)
What sort of style are you aiming for?

**Comment by aki-cat** [comment]
Yes, any help is greatly appreciated. But I personally feel bad about accepting free art from other people. As an illustrator myself I know this is bad business. So the disclaimer is: we are an open source-enthusiast group, and we only work for free on our projects. We strive only for knowledge and experience. Thus we cannot compensate you for your efforts.

If you however still would like to help, the style/theme of the game is steampunk. The characters are all anthopomorphic animals. The main character is a nonbinary cameleon and the main side characters are a woman feneco and a man black undefined feline.

**Comment by MagykalMystique** [comment]
Honestly I hadn't even thought about getting paid for my artwork hehe, mostly  because I'm only 16 and not necessarily a professional. I have more experience with humans than animals, but I would like to try expanding my horizons on this assignment. What time it'll be done by is really up in the air however... my schooling and health situation are a bit complicated at the moment. I'll most likely begin with sketches first and then ask what you think before finalizing something to outline. It'll likely be the same when I work on coloring it, as you'd most likely want to be involved in color scheme possibilities and such.

**Comment by aki-cat** [comment]
Hmm, umm, I think, actually, that we're going to pass your offer to help. You shouldn't offer free art to anyone. Furthermore, we are a college study group that is not even from your country. And, personally, I think you're too young to contribute to us without proper knowledge of who we are. If you lived in our city and could come to our reunions, that'd be fine. But as it is, it is not.

If you wanted to contribute with code, it would be fine. Code itself is not a necessarily project-decision thing. Plus, we can use pull-requests to check their viability, and even credit you properly (when using git blame). But concept art is a very central part of the project. I cannot, ethically, let anyone of whom we have no knowledge to participate in our design process.

Also, you seem to have enough on your plate. I advise you not to share too much of your personal issues with strangers, too. Especially on github, which is completely open for the general public. You should focus on local gamedev groups, or start your own. Our [game programming tutorials](https://github.com/uspgamedev/programming-tutorial) are completely free and open source. We can also recommend you tools for making your own games, like the [Godot Engine](https://godotengine.org/) or [Love2D](https://love2d.org/).

**Comment by MagykalMystique** [comment]
I understand, thank you for the advice :)
